_posts/2021-03-14-curating/curating.Rmd

```diff
@@ -0,0 +1,329 @@
+
+# About `@WeAreRLadies`
+
+The `@WeAreRLadies` rotating Twitter curator exists to "encourage and maintain Twitter engagement within the R-Ladies community", and to "spotlight female and minority genders" working with R. R-Ladies has a [comprehensive guide](https://guide.rladies.org/rocur/about/){target="_blank"} describing the program, procedures and protocols for the week, and tips for successful curation. 
+
+# Overcoming imposter syndrome
+
+You may be hesitant to sign up as a curator due to [imposter syndrome](https://caitlinhudon.com/2018/01/19/imposter-syndrome-in-data-science/){target="_blank"} - I certainly was. I was on Twitter for three years before I gathered the courage. However, you do not need to know everything about R in order to be a successful `@WeAreRLadies` curator - that is impossible! In fact,
+
+
+```{r}
+tweetrmd::include_tweet("https://twitter.com/WeAreRLadies/status/1361660625718571010")
+```
+
+Every R user, new or experienced, has a valuable perspective to share. I was particularly  impressed when [Danielle Brantley](@daniebrant){target="_blank"} excellently curated in September 2020, after being an R user for one year!
+
+My personal strategy for combating imposter syndrome is to prepare. For my curating week, my preparation involved reflecting on past curators and creating some content in advance. I hope this post helps you to prepare and motivates you to sign up. `r emo::ji("wink")`
+
+# Date selection
+
+You can view the [schedule](https://docs.google.com/spreadsheets/d/1pmphnR9EinGuAKbVqGzMIt6cWYmHNbeFVzaKJB59JdY/edit#gid=1429703967){target="_blank"} of upcoming curators to identify available dates; records of previous curators are also maintained here.  
+
+Being a curator will be time intensive, so be kind to yourself. Choose dates when you will have time to invest and a flexible work schedule. I chose Feb 15-20 
+because I hoped by then I would be recovered from an intense Q4 work cycle;
+additionally, Feb 15 (President's Day) was a company holiday. Additionally, you may
+want to select a date far enough in the future that allows you time to create
+content.
+
+Another consideration is to schedule your curation to coincide with dates that align with your interests. For example, are you passionate about Black History Month in February, LGBT Pride Month in June, or Universal Human Rights Month in December? 
+If so, take advantage of the `@WeAreRLadies` large platform as an opportunity to inform and educate others on issues that are important to you as they relate to the 
+R community.
+
+# Signing up
+
+You sign up by [submitting a form](https://docs.google.com/forms/d/e/1FAIpQLSevtboT8TNjz87qe5HZzuBHuY0aNteaZNF5e2xFQXtSeU0c0w/viewform){target="_blank"} - give yourself at least 30 minutes to sign up as part of the
```

Comment:
at this stage I wonder how much time goes between filling the form and getting some sort of confirmation from the account manager?

Reply:
excellent suggestion!

Reply:
There was a significant gap and my dates changed. I filled out the form on Oct 8 and was not re-contacted until Jan 24 - left out those details and went with something more general here: https://github.com/shannonpileggi/pipinghotdata_distill/pull/2/commits/c4194d7d163096e5af121511c8d69b68e1edaabd